src/__init__.py

```diff
@@ -44,6 +45,21 @@ def login_ghe_callback():
 	return redirect(url_for(".root"))
 
 
+@blueprint.route("/loginas/", methods=["POST"])
+def login_as():
+	"""
+	This function allows developers to be logged in as any user.
+	"""
+	if not DEV_MODE:
+		return abort(403)
```

Comment:
where is this abort defined? is it the python inbuilt abort?

Reply:
Nice catch 😅, I forgot to import it from flask. I should have tested it outside of dev mode.